spec/helpers/date_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe DateHelper do
+  describe '.format_date_of_birth' do
+    it 'formats a date from a date' do
+      expect(helper.format_date_of_birth(Date.parse('2014-07-24'))).to eq('24 July 2014')
+    end
+    it 'formats a date from a string' do
+      expect(helper.format_date_of_birth('2014-07-24')).to eq('24 July 2014')
+    end
+  end
+
+  describe '.format_date' do
+    it 'formats a day from a string' do
+      expect(helper.format_date('2014-07-24')).to eq('Thursday 24 July')
+    end
+    it 'formats a day from a date' do
+      expect(helper.format_date(Date.parse('2014-07-24'))).to eq('Thursday 24 July')
+    end
+  end
+
+  describe '.format_time_12hr' do
+    it 'formats a time from a string' do
+      expect(helper.format_time_12hr('0900')).to eq('9:00am')
+      expect(helper.format_time_12hr('1200')).to eq('12:00pm')
+      expect(helper.format_time_12hr('1545')).to eq('3:45pm')
+    end
+  end
+
+  describe '.format_time_24hr' do
+    it 'formats a time in 24 clock from a string' do
```

Comment:
Perhaps: `it 'formats a 24hr time string with a separator between hours & minutes'`